jesterTOV/inference/flows/models/nicer_maf/J06143329/maryland_3circle/submit.sh

```
@@ -0,0 +1,19 @@
+#SBATCH -p gpu_h100
+#SBATCH -t 03:00:00
+#SBATCH --gpus-per-node=1
+#SBATCH --cpus-per-gpu=1
+#SBATCH --mem-per-gpu=10G
+#SBATCH --output="../../../../../logs/%x.out"
+#SBATCH --job-name="J06143329_maryland"
+
+now=$(date)
+echo "$now"
+echo "Training flow for: J06143329_maryland"
+source /home/twouters2/projects/jester_review/jester/.venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /home/twouters2/projects/jester_review/jester/.venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)